### PR TITLE
Cody landing sign in and sign up form mobile responsive.

### PR DESCRIPTION
**Pull request by CodyCaro**
Made the forms on the landing page mobile responsive and changed the style to get rid of the widow's peak and make a bit more appealing.

### CODE REVIEW

**Review by lizdoyle — APPROVED** [review]
Changes approved but want second review to ensure this won’t clash with Aaron’s pull request

**Review by elijahdaniel — APPROVED** [review]
Code looks way cleaner!

**Pull request merged**
